### PR TITLE
add new damask notebook for different analysis

### PR DESCRIPTION
**Pull request by yangbai90**

### CODE REVIEW

File: pyiron_continuum/damask/damaskjob.py [hunk 1]
```diff
@@ -110,6 +110,12 @@ def _load_results(self, file_name="damask_loading.hdf5"):
         self.output.strain = self.average_spatio_temporal_tensors('epsilon_V^0.0(F)')
         self.output.stress_von_Mises = self.average_spatio_temporal_tensors('sigma_vM')
         self.output.strain_von_Mises = self.average_spatio_temporal_tensors('epsilon_V^0.0(F)_vM')
+    
+    def writeresults2vtk(self):
+        """
+            save results to vtk files
+        """
+        self.output.damask.export_VTK()
```
Comment (muh-hassani) [inline pyiron_continuum/damask/damaskjob.py:118]: It is desirable to export the VTK files into `self.working_directory`. There might be a workaround. You can change the current working directory to `self.working_directory` and the after exporting the vk files, return to the old working directory.

**Review by muh-hassani — CHANGES_REQUESTED** [review]
@yangbai90, thanks for your PR. I have some minor suggestions.

**Comment by yangbai90** [comment]
@muh-hassani 
Hi Muhammad, I have update the related code according to your comments, now it should work.

Best,

Yang

**Comment by muh-hassani** [comment]
@yangbai90, the notenook `damask_tutorial_elastoplasticity-powerlaw.ipynb` is failing during the tests. Can you check what the issue is? Thank you.

**Comment by yangbai90** [comment]
> @yangbai90, the notenook `damask_tutorial_elastoplasticity-powerlaw.ipynb` is failing during the tests. Can you check what the issue is? Thank you.

Hi @muh-hassani , now all the notebooks work fine on my local computer, maybe we could run the auto test  now. Thank you.

**Comment by muh-hassani** [comment]
@yangbai90, there is an error irrelevant to your implementations, which causes our test to fail. I will take a look at it later in the afternoon.  If this error is solved, I will merge it.

**Review by muh-hassani — APPROVED** [review]

**Pull request merged**